src/connection.c

```diff
@@ -153,10 +156,13 @@ static void connSocketClose(connection *conn) {
      * keep the connection until the handler returns.
      */
     if (conn->flags & CONN_FLAG_IN_HANDLER) {
+        REDIS__CONN__CLOSING(conn);
```

Comment:
I think sending the fd as part of the probe is useful

Reply:
Good catch, I was halfway to doing this